crates/anemo/src/types/mod.rs

```diff
@@ -47,8 +47,9 @@ pub mod header {
 pub enum PeerAffinity {
     /// Always attempt to maintain a connection with this Peer.
     High,
+    /// Not proactively attempt to estlish a connection but always accept inbound connection requests.
+    Allowlisted,
```

Comment:
If you'll allow me to bikeshed on this name (apologies in advance) -

an allowlist IMO is a collection of multiple allowed items

Since this applies just to a single peer I would call it "PeerAffinity::Allowed" (or maybe "PeerAffinity::AlwaysAllowed" to distinguish from the peers that are allowed only when there's space)